spec/controllers/coronavirus-form/medical_equipment_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe CoronavirusForm::MedicalEquipmentController, type: :controller do
+  let(:current_template) { 'coronavirus_form/medical_equipment' }
+  let(:session_key) { :medical_equipment }
+
+  describe 'GET show' do
+    it 'renders the form' do
+      get :show
+      expect(response).to render_template(current_template)
+    end
+  end
+
+  describe 'POST submit' do
+    let(:selected) { "Yes" }
+
+    it 'sets session variables' do
+      post :submit, params: { medical_equipment: selected }
+
+      expect(session[session_key]).to eq selected
+    end
+
+    it 'redirects to medical equipment kind' do
```

Comment:
```suggestion
    it 'redirects to medical equipment kind when yes selected' do
```

Or maybe wrap in a context block, like for 'no'?